libs/design-system/src/atoms/inputs/textAreaInputs.tsx

```diff
@@ -0,0 +1,58 @@
+import { cn } from '../../shared/classnames';
+
+export type TextAreaInputProps = {
+  /**
+   * The id of the input.
+   */
+  inputId: string;
+
+  /**
+   * Whether the input is required or not.
+   * It's false by default.
+   */
+  required?: boolean;
+
+  /**
+   * The placeholder of the input.
+   */
+  placeholder?: string;
+
+  /**
+   * The initial value of the input.
+   */
+  rows?: number;
+
+  /**
+   * Some additional class names.
+   */
+  className?: string;
+
+  /**
+   * The function that handles the change of the input.
+   */
+  handleChange: React.ChangeEventHandler<HTMLTextAreaElement>;
```

Comment:
I disagree with that one because I like the fact that `onClick`/`onChange` refer to the attributes of the underlying html tag, and that `handleAnything` refers to the actual handler (a function).

Ok with you?

Reply:
I see the point, honestly I don't have strong position on the topic so ok for me 👌 
I'll make sure to stay consistant across the repo 😉